src/ruby/puppetserver-lib/puppet/server/puppet_config.rb

```diff
@@ -23,11 +23,7 @@ def self.initialize_puppet(puppet_config)
 
     # (SERVER-410) Cache features in puppetserver for performance.  Avoiding
     # the cache is intended for agents to reload features mid-catalog-run.
```

Comment:
@cprice404 This change and a few others make me think you have a stale master branch you're working from.